src/arm/96boards.c

```diff
@@ -36,26 +36,36 @@
 
 #define DT_BASE "/proc/device-tree"
 
+#define PLATFORM_NAME_DB820C "DB820C"
 #define PLATFORM_NAME_DB410C "DB410C"
 #define PLATFORM_NAME_HIKEY "HIKEY"
 #define PLATFORM_NAME_BBGUM "BBGUM"
 #define MAX_SIZE 64
 #define MMAP_PATH "/dev/mem"
 #define DB410C_MMAP_REG 0x01000000
 
+// DB820c
+int db820c_ls_gpio_pins[MRAA_96BOARDS_LS_GPIO_COUNT] = {
+    80, 29, 124, 24, 62, 507, 10, 8, 25, 26, 23, 133,
+};
+
+const char* db820c_serialdev[MRAA_96BOARDS_LS_UART_COUNT] = { "/dev/ttyMSM0", "/dev/ttyMSM1" };
+
+// DB410c
 int db410c_ls_gpio_pins[MRAA_96BOARDS_LS_GPIO_COUNT] = {
     36, 12, 13, 69, 115, 4, 24, 25, 35, 34, 28, 33,
 };
 
 const char* db410c_serialdev[MRAA_96BOARDS_LS_UART_COUNT] = { "/dev/ttyMSM0", "/dev/ttyMSM1" };
 
+// Hikey
```

Comment:
Please either don't do, or split out such styling fixes not related to the logical change in question (adding a board support), as they make it harder to analyze the git history.

There are several such ones, I won't repeat the comment fro all of them, for brevity, but please correct them all.